### PR TITLE
Update the goyaml path to gopkg.in

### PR DESCRIPTION
**Pull request by antifuchs**
This should allow go-einhorn to operate happily in the new wonderful world of gopkg.in and moved packages!

Review plz, @zenazn?

### CODE REVIEW

File: einhorn/client.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 	"path"
 	"strings"
 
-	"launchpad.net/goyaml"
+	yaml "gopkg.in/yaml.v2"
```
Comment (zenazn) [inline einhorn/client.go]: I don't think you need the `yaml` part?

Comment (antifuchs) [inline einhorn/client.go]: Oh! You're right - I tried this before doing the renames below.

**Comment by antifuchs** [comment]
(Fixed the `yaml` name assignment)

**Comment by zenazn** [comment]
LGTM

**Pull request merged**